app/views/artisans/LevelTasksView.coffee

```diff
@@ -0,0 +1,57 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/artisans/levelTasksView'
+#ThangType = require 'models/ThangType'
+Level = require 'models/Level'
+Campaign = require 'models/Campaign'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class LevelTasksView extends RootView
+  template: template
+  id: 'level-tasks-view'
+  events:
+    'input input': 'searchUpdate'
+    'change input': 'searchUpdate'
+  excludedCampaigns = [
+    "picoctf"
+    "auditions"
+  ]
+  constructor: (options) ->
+    super options
+    @campaigns = new CocoCollection([], 
+      url: '/db/campaign?project=name,slug,tasks'
+      model: Campaign
+    )
+    @campaigns.fetch()
+    @listenTo(@campaigns, 'sync', @onCampaignsLoaded)
+    @supermodel.loadCollection(@campaigns, 'campaigns')
```

Comment:
Same here, always use collections specific to the model, `fetch` options, and `@supermodel.trackRequest` instead of `CocoCollection`, url construction or `@supermodel.loadCollection`.